docs/pages/versions/unversioned/sdk/splash-screen.mdx

```diff
@@ -29,6 +29,47 @@ Also, see the guide on [creating a splash screen image](/develop/user-interface/
 
 This example shows how to keep the splash screen visible while loading app resources and then hide the splash screen when the app has rendered some initial content.
 
+<Tabs>
```

Comment:
You'll need to add this import statement: `import { Tabs, Tab } from '~/ui/components/Tabs';` at the top of the file otherwise lack of it will break the page.